data/json/item_actions.json

```diff
@@ -2,12 +2,12 @@
   {
     "type": "item_action",
     "id": "repair_fabric",
-    "name": "Repair cotton/leather/wool/Nomex/fur"
+    "name": "Repair cotton/fur/leather/wool/Nomex"
```

Comment:
That's just another meaningless work for translators.

Reply:
I know now :unamused:. I'd love to have it changed for English if only for the sake of consistency, but I hate the fact that every change in English file removes the translations from another languages, especially in cases such as these where each language. I can handle Polish as I am know registered in Transifex, but to help with other languages I'd have to join all the languages plus I'd have to remember to copy all respective strings beforehand (doable but definitely PITA).

I'll write an issue report, so maybe something could be done to at least mitigate those in the future, and will address #24761 first, as there's way more strings to be concerned here.